ci/jenkins/templates/gpu_jenkinsfile.groovy.j2

```diff
@@ -74,6 +74,14 @@
     label: 'Run Java unit tests',
   )
   {% endif %}
+  sh (
+    script: "${docker_run} ${ci_gpu} python3 ./tests/scripts/task_build.py --sccache-bucket tvm-sccache-prod --cmake-target opencl-cpptest --build-dir build",
+    label: 'Make OpenCL cpp unit tests',
+  )
+  sh (
+    script: "${docker_run} ${ci_gpu} ./tests/scripts/task_opencl_cpp_unittest.sh",
+    label: 'Run OpenCL cpp unit tests',
+  )
```

Comment:
Can you move this next to the `cpp_unittests` above? Right now this job is run 3 times afaik and pytest knows that each job should only run 1/3 of the tests, but this runs all the OpenCL tests 3 times

Reply:
Done